### PR TITLE
feat: add snapshot verifier

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: paket.lock [hunk 1]
```diff
@@ -429,6 +520,7 @@ NUGET
       System.Runtime (>= 4.3)
     System.Diagnostics.DiagnosticSource (6.0)
       System.Runtime.CompilerServices.Unsafe (>= 6.0)
+    System.Diagnostics.EventLog (7.0)
```
Comment (jvandaal) [inline paket.lock:523]: Should we prevent this from being 7.0 by adding strategy: min to paket.dependencies?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -116,6 +116,7 @@ jobs:
         node packages/Be.Vlaanderen.Basisregisters.Build.Pipeline/Content/ci-nuget.js dist/$REPONAME/$COMMANDHANDLING.$SEMVER.nupkg
         node packages/Be.Vlaanderen.Basisregisters.Build.Pipeline/Content/ci-nuget.js dist/$REPONAME/$COMMANDHANDLING.Idempotency.$SEMVER.nupkg
         node packages/Be.Vlaanderen.Basisregisters.Build.Pipeline/Content/ci-nuget.js dist/$REPONAME/$COMMANDHANDLING.SqlStreamStore.$SEMVER.nupkg
+        node packages/Be.Vlaanderen.Basisregisters.Build.Pipeline/Content/ci-nuget.js dist/$REPONAME/SnapshotVerifier.$SEMVER.nupkg
```
Comment (ArneD) [inline .github/workflows/release.yml:119]: this won't work, there's no Be.Basisregisters..

File: src/Be.Vlaanderen.Basisregisters.SnapshotVerifier/SnapshotVerifier.cs [hunk 3]
```diff
@@ -0,0 +1,196 @@
+namespace Be.Vlaanderen.Basisregisters.SnapshotVerifier
```
Comment (ArneD) [inline src/Be.Vlaanderen.Basisregisters.SnapshotVerifier/SnapshotVerifier.cs:1]: Add some Information logging, like starting, stopping, verifying id ...

**Review by ArneD — CHANGES_REQUESTED** [review]
Missing `paket.template`

File: .github/workflows/release.yml (hunk 2 above)
Comment (emalfroy) [reply .github/workflows/release.yml:119]: What does it needs to be then?

Comment (ArneD) [reply .github/workflows/release.yml:119]: What you put in the `paket.template` as `id`

**Pull request merged**